.github/workflows/homebrew-assets-ci.yml

```diff
@@ -0,0 +1,189 @@
+name: Homebrew Assets CI
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/homebrew-assets-ci.yml
+      - utils/convertkit/assets-convert-homebrew.py
+  pull_request:
+    branches:
+      - main
+    paths:
+      - .github/workflows/homebrew-assets-ci.yml
+      - utils/convertkit/assets-convert-homebrew.py
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: ${{ matrix.source.name }} for ${{ matrix.target.name }}
+    runs-on: ubuntu-latest
+    container: ${{ matrix.target.docker_image }}
+    env:
+        DEVKITPRO: /opt/devkitpro
+    strategy:
+      fail-fast: false
```

Comment:
If this set to `false`, that means, if any job will fail, the chain will continue. That means, thing would turn into the mess if one job depends on a previous.